### DISCUSSION TITLE
Feature Request: Subcategory Creation in Github Discussions 🔥

### DISCUSSION THREAD

**Discussion by Irisation23** [community/community#51288] [Discussions]
### Select Topic Area

Product Feedback

### Body

Hello. I would like to request the ability to create subcategories within Github Discussions. Currently, only general categories can be created, but it would be helpful to have the ability to create `subcategories under a parent category`. 

This would allow for more organized and structured conversations. 
Please consider this feature. Thank you. 🙏🏻

**Comment by LindseyB** [comment]
This is coming. Stay tuned.

**Comment by jsoref** [comment]
It arrived weeks after you asked... https://github.com/orgs/community/discussions/32323#discussioncomment-8226325
